pvlive_api/pvlive.py

```diff
@@ -40,21 +42,29 @@ def __init__(self, retries=3):
         self.max_range = {"national": timedelta(days=365), "regional": timedelta(days=30)}
         self.retries = retries
 
-    def latest(self, pes_id=0, extra_fields=""):
+    def latest(self, pes_id=0, data_frame=False, extra_fields=""):
         """
         Get the latest PV_Live generation result from the API.
 
         Parameters
         ----------
         `pes_id` : int
             The numerical ID of the PES region of interest. Defaults to 0 (i.e. national).
+        data_frame` : boolean
+            Boolean indicator whether to return data as a dataframe or not.
         `extra_fields` : string
             Comma-separated string listing of the names of any extra fields required.
         Returns
         -------
         tuple
             Tuple containing the pes_id, datetime_GMT and generation_MW fields of the latest
             PV_Live result, plus any extra_fields in the order specified.
+            
+            OR
+            
```

Comment:
Doesn't conform to [NumpyDoc format](https://numpydoc.readthedocs.io/en/latest/format.html) so will likely break Sphinx Autodoc